regression/basic_flow.py

```diff
@@ -0,0 +1,165 @@
+# Copyright 2023 MosaicML Streaming authors
```

Comment:
Sorry, I did not think a proper name when I created a task. Can we rename this file to `iterate_data.py` or something better ?